migrations/2024.06.13T13.52.01.bundles.nullable-parent-and-root.sql

```diff
@@ -0,0 +1,47 @@
+DROP TABLE new_data_items;
+
+CREATE TABLE new_data_items (
```

Comment:
will this effectively reset each node's `new_data_items` when the migration performs?

Reply:
Yes

Reply:
Yeah, it does, but it shouldn't be a big deal. They'll just pick up and start importing again at the stable threshold. Also we have bundle import retry mechanisms that should kick in if anything is missed.